pkg/storaged/dialog.jsx

```diff
@@ -1048,7 +1048,7 @@ export const BlockingMessage = (usage) => {
                           columns={[
                               { title: _("Device") },
                               { title: _("Location") },
-                              { title:  _("Use") }
+                              { title: _("Use") }
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221109-102136-9fe18085-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)